pkg/test/useraccount/useraccount_test.go

```diff
@@ -1,19 +1,17 @@
-package useraccount_test
+package useraccount
```

Comment:
this is not related to dependencies.
unless I'm missing some background context, I didn't see any value in having a separate package name for useraccount_test. It would be following convention to have test file in the same package.

Reply:
+1 , unless it's testing internal methods, usually tests should be in the `_test` package.

Reply:
gotcha, thanks! reverted in [fe45e4a](https://github.com/codeready-toolchain/toolchain-common/pull/449/commits/fe45e4a438635f444bc63a8485acdaed0ed231a6)
You're right, I checked the other packages in the repo too!